crates/cli/tests/cli_run.rs

```diff
@@ -411,7 +439,7 @@ mod cli_run {
             "helloWorld.roc",
             "helloWorld",
             "Hello, World!\n",
-            true,
+            UseValgrind::Yes,
```

Comment:
I wasn't sure at first about going from booleans to types but after seeing it in a function, it is indeed nicer :)